app/code/Magento/Theme/view/frontend/web/css/tabs.css

```diff
@@ -1,49 +0,0 @@
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-div.ui-tabs {
```

Comment:
I think it's related to https://devdocs.magento.com/guides/v2.4/javascript-dev-guide/widgets/widget_tabs.html.

And still used in https://github.com/magento/magento2/blob/7c6b6365a3c099509d6f6e6c306cb1821910aab0/app/code/Magento/Ui/view/base/templates/layout/tabs/nav/default.phtml#L11
https://github.com/magento/magento2/blob/7c6b6365a3c099509d6f6e6c306cb1821910aab0/lib/web/jquery/ui-modules/tabs.js#L60

Reply:
The same with this file, it doesn't seem to be loaded anywhere. But I'm still double checking this.

Reply:
@ihor-sviziev, I couldn't find any style issue, that may be related to the styles from the removed files.